lua/tabout/node.lua

```diff
@@ -37,8 +37,8 @@ M.get_node_at_cursor = function(dir)
     local line = cursor[1] - 1
     local col = dir == 'backward' and cursor[2] and cursor[2] - 1 or cursor[2]
     local cursor_range = {line, col, line, col}
-    local parser = vim.treesitter.get_parser(0)
-    if parser then
+    local ok, parser = pcall(vim.treesitter.get_parser, 0)
+    if ok and parser then
       parser:parse()
     else
         logger.debug("get_node_at_cursor: No parser found for filetype " .. vim.bo[0].filetype)
```

Comment:
In the else node we should do two things:
- notify the user about the error
-- `vim.notify` "no parser found for filetype xxx"
- fire a normal tab press event
-- sth like `nvim_feedkeys` and then based on dir either <TAB> or <S-TAB>

Reply:
notify the user about the error
-> replaced logger.debug with vim.notify, added skip_notify map to avoid too much noise.

fire a normal tab press event
->  When it returns nothing in this function, caller already does normal tab press event. Do we need other logic?

Reply:
Hi @aca,

thanks for the update 👍 

Added some comments and you are right about the normal tab event, thanks for pointing that out 😊